tmt/utils.py

```diff
@@ -2577,3 +2579,65 @@ def find_fmf_root(path: str) -> List[str]:
         raise MetadataError(f"No fmf root present inside '{path}'.")
     fmf_roots.sort(key=lambda path: len(path))
     return fmf_roots
+
+
+#
+# JSON schema-based validation helpers
+#
+# Aims at FMF data consumed by TMT, but can be used for any structure.
+#
+
+# `Schema` represents a loaded JSON schema structure. It may be fairly complex,
+# but it's not needed to provide the exhaustive and fully detailed type since
+# TMT code is not actually "reading" it. Loaded schema is passed down to
+# jsonschema library, and while `Any` would be perfectly valid, let's use a
```

Comment:
@happz seems unfinished sentence with the one below

Reply:
Fixed.